src/main/java/Event.java

```diff
@@ -0,0 +1,13 @@
+public class Event extends Task {
+    protected String eventAt;
```

Comment:
Should this variable mention something about time? eventAt may have two meanings 1) location of the event 2) time of the event.